### PR TITLE
361 Tags Add-Edit-Delete functionality

### PR DESCRIPTION
**Pull request by petark7**
This is a pull request for the Tags page functionality. With the current implementation, the following features are available:

1. Adding Tags (with validation)
2. Deleting Tags
3. Editing tags (with validation)

Related tickets:

[Edit tag](https://github.com/orgs/learnhubmk/projects/2/views/10?pane=issue&itemId=67577782)
[Delete tag](https://github.com/orgs/learnhubmk/projects/2/views/10?pane=issue&itemId=67592184)
[Add tag](https://github.com/orgs/learnhubmk/projects/2/views/10?pane=issue&itemId=67574996)

### CODE REVIEW

**Review by stoimenov95 — CHANGES_REQUESTED** [review]
Hey @petark7, great job on the CRUD operations regarding the tags. I successfully tested the feature, at least the first part of it. I wasn't able to search through the existing tags, and I am not sure if I should be able to apply tags to blog posts.

Can you please also check if we should have the validation messages in Macedonian?

**Comment by stoimenov95** [comment]
@petark7 please resolve the merge conflicts before submitting for the next round of review.

File: app/content-panel/tags/page.tsx [hunk 1]
```diff
@@ -1,15 +1,108 @@
...
+    { id: '1', name: 'React' },
+    { id: '2', name: 'TypeScript' },
+    { id: '3', name: 'NextJS' },
+  ]);
+  const [showAddTag, setShowAddTag] = useState(false);
+  const [editingTagId, setEditingTagId] = useState<string | null>(null);
+
+  const validationSchema = Yup.object().shape({
+    tagName: Yup.string()
+      .required('Tag name is required')
```
[the remark this answers is not in the document]
Comment (petark7) [reply app/content-panel/tags/page.tsx]: I think you are correct - we should have those in Macedonian and will be promptly updated.

Regarding the search feature, it wasn't requested in any of the tickets hence why I haven't implemented it but if needed I can implement it here as well.

As far as I know, currently we are not applying the tags to blog posts.

[the remark this answers is not in the document]
Comment (edichoska) [reply app/content-panel/tags/page.tsx]: Lets create a ticket for  the search functionality. 
We will be adding tags in blog posts. That is on another ticket currently in review by jasmina

[the remark this answers is not in the document]
Comment (edichoska) [reply app/content-panel/tags/page.tsx]: @petark7 Do we need this PR?
I saw you have created separate PRs toward the base branch: [feature/tags-base] (I already approved those)
I assume after you merge those, you are going to create a PR from `feature/tags-base` towards `staging`

[the remark this answers is not in the document]
Comment (petark7) [reply app/content-panel/tags/page.tsx]: @edichoska That is correct. This PR here was created due to misunderstanding and combined all the features at once which is not useful and can therefore be disregarded.

**Comment by edichoska** [comment]
Closing this PR because it is a duplicate

**Pull request closed without merging**